A3A/addons/ultimate/CfgFunctions.hpp

```diff
@@ -35,6 +35,26 @@ class CfgFunctions
             class emission {};
             class fillMapAnomalies {};
         };
+        class D37Dome
+        {
+            file = QPATHTOFOLDER(functions\D37-Dome);
```

Comment:
```suggestion
            file = QPATHTOFOLDER(functions\D37_dome);
```
Rename the folder too